instructions/docs/getting-started.md

```diff
@@ -4,7 +4,7 @@
 
 In this section, you will prepare your AWS Environment to work with the Busy Engineer's Document Bucket. As you work through the workshop, you will gain familiarity with the core concepts required to use AWS KMS, the AWS Encryption SDK, and how to leverage features like multiple CMK support and Encryption Context to secure your application.
 
-To prepare your environment, you will do the following:
+In this exercise, we will walk you through setting up your environment, step by step. At the end of this exercise you will have completed the following:
 
 1. Deploy a CloudFormation stack to launch a Cloud9 IDE
```

Comment:
consider using bullets instead of numbers here. The numbers strongly imply that I should start doing things.

Reply:
Oooh that is helpful to know. Will change.